edxval/pacts/utils.py

```diff
@@ -0,0 +1,129 @@
+"""
+Utility methods needed during pact verification.
+"""
+
+from edxval.models import CourseVideo, EncodedVideo, Profile, Video, VideoImage, VideoTranscript
+from edxval.tests import constants
+from edxval.views import VideoDetail, VideoImagesView, VideoStatusView, VideoTranscriptView
+
+VIDEO_ID = "3386eda1-8d6e-439d-tb89-e90a5c52h103"
+COURSE_ID = "course-v1:testX+test123+2030"
+DEFAULT_PROFILE_NAMES = ['desktop_mp4', 'mobile_low', 'mp3', 'hls']
```

Comment:
Is there a single source for this list?

Reply:
In migration https://github.com/edx/edx-val/blob/master/edxval/migrations/0001_squashed_0016_add_transcript_credentials_model.py#L18. and it has some extra profiles not used by VEM.